README.md

```diff
@@ -2,24 +2,56 @@
 
 > Manage Java projects in Visual Studio Code
 
-[![GitHub Actions](https://img.shields.io/github/workflow/status/microsoft/vscode-java-dependency/CI/master?style=flat-square)](https://github.com/microsoft/vscode-java-dependency/actions?query=workflow%3ACI+branch%3Amaster)
+[![GitHub Actions](https://img.shields.io/github/workflow/status/microsoft/vscode-java-dependency/CI/main?style=flat-square)](https://github.com/microsoft/vscode-java-dependency/actions?query=workflow%3ACI+branch%3Amain)
 
 ## Overview
 
 A lightweight extension to provide additional Java project explorer features. It works with [Language Support for Java by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java) to provide the following features:
 
-* Project View
+### Project View
 
-![project-view](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/master/images/project-explorer.png)
+![project-view](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/main/images/project-explorer.png)
 
-* Create Java Projects
+### Create Java Projects
 
-![create project](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/master/images/create-project.png)
+![create project](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/main/images/create-project.png)
 
-* Export Jar
+### Export Jar
 > Note: For Spring Boot projects, please use the build tool to build the executable jar, for example: `mvn package`.
 
-![export jar](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/master/images/export-jar.png)
+![export jar](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/main/images/export-jar.png)
+
+### Manage Dependency
+
+You can work with JAR files directly without any build tools. Go to `JAVA PROJECTS` view, find the `Referenced Libraries` node and click the `+` icon:
+
+![Reference JAR Files](https://raw.githubusercontent.com/Microsoft/vscode-java-dependency/main/images/manage-dependency.gif)
```

Comment:
Can we make the gif more neat? For example, eliminate the un-relevant frames like changing the panel size.